cliquet/__init__.py

```diff
@@ -16,6 +16,13 @@
 from cliquet.logs import logger
 
 
+import warnings
+warnings.warn("cliquet is now deprecated. Please update to kinto.core at "
```

Comment:
I wasn't sure because the package name is `cliquet`, but the library's name is `Cliquet`.